source/_static/css/mattermost-global.css

```
@@ -143,42 +146,37 @@ details.mm-sbom > ul > li {
 
 
 .notification-bar.closed {
     display: none;
 }
 
 .notification-bar__content {
-    padding: 0.5rem 3rem 0.5rem 1rem;
+    display: flex;
+    align-items: center;
+    padding: 0 1rem;
     font-family: "trade-gothic-next", "Lato", "Helvetica Neue", "Arial", sans-serif;
```

Stylelint (17.9.0)

[error] Expected no quotes around "trade-gothic-next" (font-family-name-quotes)

(font-family-name-quotes)